packages/dataregistry-csvviewer-extension/src/text_to_csv.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license BSD-3-Clause
+ *
+ * Copyright (c) 2019 Project Jupyter Contributors.
+ * Distributed under the terms of the 3-Clause BSD License.
+ */
+
+import { createConverter } from "@jupyterlab/dataregistry";
+import datatypes from "./datatypes"; // FIXME
+
+/**
+ * Interface describing an object containing data to convert.
+ *
+ * @private
+ */
+interface Data {
```

Comment:
Again here, since the functions are not inline you have to manually annotate them with types, and this type is too general.